feature-libs/order/assets/translations/translations.ts

```diff
@@ -20,5 +20,9 @@ export const orderTranslationChunksConfig: TranslationChunksConfig = {
     'returnRequest',
     'reorder',
   ],
-  MyAccountV2Order: ['myAccountV2OrderHistory', 'myAccountV2OrderDetails'],
+  MyAccountV2Order: [
```

Comment:
[MEDIUM]
For consistency reasons, let's keep translation keys and chunk names camelCase
```suggestion
  myAccountV2Order: [
```

Reply:
done